florist/app/jobs/hooks.tsx

```diff
@@ -8,3 +10,40 @@ export function useGetJobsByJobStatus(status: string) {
     });
     return { data, error, isLoading };
 }
+
+export function useGetModels() {
+    return useSWR("/api/server/models", fetcher);
+}
+
+export function useGetClients() {
+    return useSWR("/api/server/clients", fetcher);
+}
+
+export const usePost = () => {
```

Comment:
Very cool! I don't know if this is a common pattern but I liked the way you set this up

Reply:
Yes, I tried multiple ways of doing it and found this in a medium post which appeared to be the most elegant way. I didn't know about hooks before I started working with Next.js (they were not a thing the last time I worked with React in 2017) and it's growing on me as a concept.